estela-web/src/utils.ts

```diff
@@ -25,16 +25,28 @@ export function convertDateToString(date: Date | undefined): string {
     return "";
 }
 
-export function formatSecondsToHHMMSS(seconds: number): string {
-    const hours = Math.floor(seconds / 3600);
-    const minutes = Math.floor((seconds % 3600) / 60);
-    const remainingSeconds = Math.round(seconds) % 60;
-    const formattedTime = [
-        hours.toString().padStart(2, "0"),
-        minutes.toString().padStart(2, "0"),
-        remainingSeconds.toString().padStart(2, "0"),
-    ].join(":");
-    return formattedTime;
+export function formatSecondsToHHMMSS(seconds: number | string): string {
+    if (typeof seconds === "number") {
```

Comment:
I think this makes things ambiguous. The `seconds` parameter should only be a number. If we know it is a string already formatted as `HH:MM:SS`, then we shouldn't pass it through this function.

Reply:
I miss to remove it.